package-lock.json

```diff
@@ -22092,7 +22092,7 @@
         },
         "got": {
           "version": "9.6.0",
-          "resolved": "https://registry.npmjs.org/got/-/got-9.6.0.tgz",
+          "resolved": "http://registry.npmjs.org/got/-/got-9.6.0.tgz",
```

Comment:
I suspect the package lock change here was accidental, I don't think this PR should do this

Reply:
fixed